dist/win/build.ps1

```diff
@@ -35,7 +35,7 @@ Write-Output "`$buildDir=$buildDir"
 Write-Output "`$Env:JAVA_HOME=$Env:JAVA_HOME"
 
 $copyright = "(C) $CopyrightStartYear - $((Get-Date).Year) $Vendor"
-$loopbackAlias = 'cryptomator-vault'
+$loopbackAlias = "$AppName-vault"
```

Comment:
This leads to odd behaviour in whitelabels. Such as "Brand-Tresor-vault". Never try to be "smart" when parameterizing. There is no pain in defining multiple parameters with similar values.

Reply:
Fixed in 1b0eb34734dc190bc0b2e9454c1c0cfc86b82a93